tests/modules/jmeter/test_JMeterExecutor.py

```diff
@@ -2619,6 +2619,33 @@ def test_jsr223_multiple(self):
         self.assertEqual(None, pre.find(".//stringProp[@name='parameters']").text)
         self.assertEqual('vars.put("a", 1)', pre.find(".//stringProp[@name='script']").text)
 
+    def test_jsr223_scenario_level_block(self):
+        script = RESOURCES_DIR + "/jmeter/jsr223_script.js"
+        self.configure({
+            "execution": {
+                "scenario": {
+                    "jsr223": {
+                        "language": "javascript",
+                        "script-file": script,
+                        "parameters": "first second"
+                    },
+                    "requests": [{
+                        "url": "http://blazedemo.com/",
+                    }]
+                }
+            }
+        })
+        self.obj.prepare()
+        xml_tree = etree.fromstring(open(self.obj.modified_jmx, "rb").read())
+        post_procs = xml_tree.findall(".//JSR223PostProcessor[@testclass='JSR223PostProcessor']")
```

Comment:
Shouldn't we check jsr is faced exactly on threadgroup level? Would be great to see the difference between 'old' and 'new' levels of jsr generation.